com.borgbase.vorta.yaml

```diff
@@ -0,0 +1,90 @@
+app-id: com.borgbase.vorta
+
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+
+command: vorta
+
+finish-args:
+  # X11/Wayland
+  - --socket=x11
+  - --share=ipc
+  - --socket=wayland
+  - --device=dri
+  # Sync
+  - --share=network
+  # Filesystem access
+  - --filesystem=home
+  - --filesystem=/run/media
+  - --filesystem=/media
+  # dconf access
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  # Tray icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
```

Comment:
You just need 
```
      "--talk-name=org.kde.StatusNotifierWatcher"
```
for the tray icon

Reply:
Unfortunately it is not that way.
Without "own-name" I get a failed-to-register-"StatusNotifierItem-2-1" or "StatusNotifierItem-4-1" error.

Reply:
with `log-session-bus` I get:
Filtering message due to arg0 org.freedesktop.Notifications, policy: 0 (required 1)
Filtering message due to arg0 org.kde.StatusNotifierItem-2-1, policy: 2 (required 3)